gsoc/suborg_guidelines.md

```diff
@@ -56,21 +58,21 @@ To add your own, you will have to create a pull-request to [our repository][OA r
 following the [template](https://github.com/OpenAstronomy/openastronomy.github.io/blob/master/_projects/_template.md).
 Take a look at [how others have done it before](https://github.com/OpenAstronomy/openastronomy.github.io/pull/182).
 The admins and other members will review your ideas and provide feedback. Once
-done they will be merged and displayed on our website.
+done, they will be merged and displayed on our website.
 
 It's also possible that a student comes up with their own ideas! That's OK, the
-only limitation is that either the student or the admin will have to find a 
+only limitation is that either the student or the admin will have to find a
 mentor for that idea who is familiar with the programming language, the repository,
 and the idea in itself.
 
 ### Mentors
 
-For each project idea you need to have two mentors that are committed to participate
+For each project idea you need to have at least **two** mentors that are committed to participate
 in the programme. The mentors need most importantly be familiar with the repository,
 normally they are real core-contributors, they need to be familiar with the idea and
 have some idea on how that could be implemented.
 
-The mentors also need to have time for this!!! a lot of time!! We expect around 
+The mentors also need to have time for this!!! a lot of time!! We expect around
```

Comment:
This is a bit too informal for my taste, and a whole lot more exclamation marks than comfortable 😅  
 but ok to leave as is for now

Reply:
haha, makes sense